packages/accounts/src/light-account/types.ts

```diff
@@ -0,0 +1,6 @@
+import { z } from "zod";
+import { LightAccountProviderConfigSchema } from "./schema.js";
+
+export type LightAccountProviderConfig = z.infer<
```

Comment:
```suggestion
export type LightAccountProviderConfig = z.input<
```

Reply:
this leads to errors for me:
<img width="946" alt="image" src="https://github.com/alchemyplatform/aa-sdk/assets/43521356/faf16011-897f-45cb-aa05-625cc2b78266">